src/services/api-service.ts

```diff
@@ -72,36 +92,23 @@ class ApiService {
   }
 
   async sendQueryRequest(query: string, props: any, structuredLogInfo?: StructuredLogInfo): Promise<any> {
-    if (this.logger.structuredLog !== undefined && structuredLogInfo !== undefined) {
-      this.logger.structuredLog(
-        structuredLogInfo.network,
-        `Sending query request ${query} with props ${JSON.stringify(props)}`,
-        structuredLogInfo.tagId,
-      )
-    } else {
-      this.logger.log(`Sending query request ${query} with props ${JSON.stringify(props)}`)
-    }
-
+    this.logInfo(`Sending query request ${cleanRequest(query)} with props ${JSON.stringify(props)}`, structuredLogInfo)
     try {
-      return await this.client.request(query, props)
+      return await this.client.rawRequest(query, props)
     } catch (error: any) {
-      if (this.logger.structuredLogError !== undefined && structuredLogInfo !== undefined) {
-        this.logger.structuredLogError(structuredLogInfo.network, error, [
-          ...(structuredLogInfo.tagId as (string | number)[]),
-          this.errorColor(`Error sending query request`),
-        ])
-      } else {
-        this.logger.error(`Error sending query request ${error}`)
-      }
+      this.logError('Error sending query request', error, structuredLogInfo)
     }
   }
 
-  async sendMutationRequest(mutation: string, props: any): Promise<any> {
-    this.logger.log(`Sending mutation request ${mutation} with props ${JSON.stringify(props)}`)
+  async sendMutationRequest(mutation: string, props: any, structuredLogInfo?: StructuredLogInfo): Promise<any> {
```

Comment:
@BravoNatalie Please set the return type so we no longer use any

Reply:
That is probably going to be more involved because this is a generic mutation function that hits multiple resources. It's part of a larger refactor I was thinking of doing post mainnet

Reply:
Basically we will need typed responses and requests for each model

Reply:
@alexanderattar I think we can at least create the shape of the return right? This fn return is `this.client.rawRequest(mutation, props)` which returns `{ data, errors, extensions, headers, status }` We should be able to have at least that much visibility right?

Reply:
Okay let's try that to start. Maybe a RawResponse interface? Worth checking if there's one we can already import from the graphql library @BravoNatalie let us know if we can help!